scipy/stats/mstats_basic.py

```diff
@@ -1747,7 +1747,7 @@ def winsorize(a, limits=None, inclusive=(True, True), inplace=False,
         indicate an open interval.
     inclusive : {(True, True) tuple}, optional
         Tuple indicating whether the number of data being masked on each side
-        should be rounded (True) or truncated (False).
```

Comment:
You should make this fix in a separate PR

Reply:
Sorry for committing this in the same PR(I hadn't created a separate branch so, it got pushed into the same PR). I reverted this commit and will create a separate PR.

Reply:
I am closing this PR and will make two separate pull requests.